rtos/source/TARGET_CORTEX/mbed_rtx_storage.h

```diff
@@ -42,6 +42,8 @@ extern "C" {
  This file breaks abstraction layers and uses RTX internal types, but it limits the contamination to single, RTOS
  implementation specific, header file, therefore limiting scope of possible changes.
   @{
+
+  @note Baremetal profile: This rtos storage is not supported.
```

Comment:
It does not need to be documented here as these are RTOS primitives storage types for RTX.